src/iotconnect_telemetry.c

```diff
@@ -266,16 +267,16 @@ bool iotcl_telemetry_set_null(IotclMessageHandle message, const char *path) {
 const char *iotcl_create_serialized_string(IotclMessageHandle message, bool pretty) {
     const char *serialized_string = NULL;
     if (!message) {
-        PRINTF("message is NULL\n");
+        fprintf(stderr, "message is NULL\n");
```

Comment:
The iotc-c-lib does not printf. One reason is that the system using it may simply not have printf. Oher is that it is up to the SDK to deal with the errors. There are also several other reasons (like do you do \r\n on printf or just \n?) and I can explain them if needed.

The printf messages need to be removed.